packages/api-fetch/src/middlewares/theme-preview.js

```diff
@@ -1,32 +1,39 @@
 /**
  * WordPress dependencies
  */
-import { addQueryArgs, hasQueryArg } from '@wordpress/url';
+import { addQueryArgs, getQueryArg, removeQueryArgs } from '@wordpress/url';
 
 /**
  * This appends a `wp_theme_preview` parameter to the REST API request URL if
  * the admin URL contains a `theme` GET parameter.
  *
+ * If the REST API request URL has contained the `wp_theme_preview` parameter,
+ * then bypass this middleware.
```

Comment:
Would it be nice to update the comment, given that we only expect empty characters?

```suggestion
 * If the REST API request URL has contained the `wp_theme_preview` parameter as `''`,
 * then bypass this middleware.
```